core/tests/puppeteer-acceptance-tests/user-utilities/super-admin-utils.ts

```diff
@@ -43,8 +43,12 @@ export class SuperAdmin extends BaseUser {
         allRoles[i]
       );
       if (roleText.toLowerCase() === role) {
-        await allRoles[i].click();
-        await this.page.waitForNetworkIdle();
+        // Here we are clicking on the role in the DOM since the options
+        // might not be in view.
+        await Promise.all([
+          allRoles[i].evaluate(role => (role as HTMLElement).click(), role),
```

Comment:
I am a bit confused about what this is doing, sorry.

`role` is a string, right (per the method signature at the top)? If so then how can we interpret it as a clickable html element?

If it's a local variable then I suggest we name allRoles differently so that the name describes an element on the page and not a string (which "role" does).

Reply:
Done, meant to have "element"